src/UWPClientLibrary/Microsoft.FactoryOrchestrator.UWPClient.csproj

```diff
@@ -23,7 +22,7 @@
     <TargetName>Microsoft.FactoryOrchestrator.UWPClient</TargetName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17763.0</TargetPlatformVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>

```

Comment:
be sure to update the readme if it refers to 17763

Reply:
yep it does :), suggest you fix it in this PR

Reply:
Done!